INSTALL.rst

```diff
@@ -1,156 +1,230 @@
+
+
+Installation via pip and conda
+------------------------------------------------------------------------------
+
+These install only ``scikit-image`` and its dependencies; pip has an option to
+include related packages.
+
+.. _install-via-pip:
+
+pip
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prerequisites to a pip install: You're able to use your system's command line to
+install packages and are using a
+`virtual environment \
+<https://towardsdatascience.com/virtual-environments-104c62d48c54?gi=2532aa12906#ee81>`_
+(any of
+`several \
+<https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe>`_\
+).
+
+Nothing will stop you from using pip without a virtual environment,
+but `it's a bad idea that will bite you later \
+<https://en.wikipedia.org/wiki/Dependency_hell>`_.
 
-We are assuming that you have default Python environment already configured on
-your computer and you intend to install ``scikit-image`` inside of it. If you
-want to create and work with Python virtual environments, please follow the
-instructions on `venv`_ and `virtual environments`_.
+To install the current ``scikit-image`` you'll need at least Python 3.6. If
+your Python is older, pip will find the most recent compatible version.
 
-There are two ways you can install ``scikit-image`` on your preferred Python
-environment.
+.. code-block:: sh
 
-1. Standard Installation
-2. Development Installation
+  # Update pip
+  python -m pip install -U pip
+  # Install scikit-image
+  python -m pip install -U scikit-image
 
-1. Standard Installation:
--------------------------
+To include a selection of other scientific python packages as well,
+replace the last line with
 
-``scikit-image`` comes pre-installed with several Python distributions,
-including `Anaconda <https://www.anaconda.com/distribution/>`_,
-`Python(x,y) <https://python-xy.github.io/>`_ and
-`WinPython <https://winpython.github.io/>`_.
+.. code-block:: sh
 
-On all major operating systems, install it via shell/command prompt::
+    python -m pip install -U scikit-image[optional]
 
-  pip install scikit-image
 
-If you are running Anaconda or miniconda, use::
+.. _install-via-conda:
 
-  conda install -c conda-forge scikit-image
+conda
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The wheels can be downloaded manually from `PyPI <https://pypi.org/project/scikit-image/#files>`__.
+Miniconda is a bare-essentials version of the Anaconda package; you'll need to
+install packages like ``scikit-image`` yourself. Like Anaconda, it installs
+Python and provides virtual environments.
 
-2. Development Installation:
-----------------------------
+- `conda documentation <https://docs.conda.io>`_
+- `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+- `conda-forge <https://conda-forge.org>`_, a conda channel maintained
+  with the latest ``scikit-image`` package
 
```

Comment:
Did we want to provide the conda command here?

Reply:
Again, please feel free to clarify.